File: Tugas coding modul 5/coding c/PRAK503-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

int maksimal(int a, int b){
    int x = a;
    if (b > a){
        x = b;
    }

    return x;
}

int minimal(int a, int b){
    int y = a;
    if (b < a){
        y = b;
    }

    return y;
}

int main(){
    int batas = 0;
    int maks = -100000; // hanya sebagai inisiasi
    int minim = 100000; // hanya sebagai inisiasi
    int bilangan;

    scanf("%d", &bilangan);
    while(batas < bilangan){
        int nilai;
        scanf("%d", &nilai);
        maks = maksimal(maks, nilai);
        minim = minimal(minim, nilai);
        batas++;
    }
    
    printf("%d %d", maks, minim);
}

File: Tugas coding modul 5/coding py/PRAK503-2310817120005-Noviana_Nur_Aisyah.py
def maksimal(a, b):
    x = a
    if b > a:
        x = b
    return x

def minimal(a, b):
    y = a
    if b < a:
        y = b
    return y

def main():
    batas = 0
    maks = -100000
    minim = 100000
    bilangan = int(input())
    nilai = list(map(int, input().split()))

    for i in range(batas, bilangan):
        maks = maksimal(maks, nilai[i])
        minim = minimal(minim, nilai[i])

    print(f"{maks} {minim}")

main()